File: app/src/main/java/com/example/mobileproject/holder/CommentItemHolder.java
package com.example.mobileproject.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.example.mobileproject.R;

public class CommentItemHolder  extends RecyclerView.ViewHolder {
    public TextView nickname;
    public TextView contents;


    //public ImageView imageView;
    //public Button share;
    //public Button more;

    public CommentItemHolder(View itemView) {
        super(itemView);
        nickname = (TextView) itemView.findViewById(R.id.comment_nickname_textview);
        contents = (TextView) itemView.findViewById(R.id.comment_contents_textview);

        //imageView = itemView.findViewById(R.id.imageView);
    }
}

File: app/src/main/java/com/example/mobileproject/holder/CommentItemHolder.kt
package com.example.mobileproject.holder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import com.example.mobileproject.R

class CommentItemHolder
//public ImageView imageView;
//public Button share;
//public Button more;

(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var nickname: TextView = itemView.findViewById<View>(R.id.comment_nickname_textview) as TextView
    var contents: TextView = itemView.findViewById<View>(R.id.comment_contents_textview) as TextView

    init {

        //imageView = itemView.findViewById(R.id.imageView);
    }
}
